Dockerfile

```diff
@@ -94,6 +94,7 @@ RUN export PATH="/opt/miniconda-latest/bin:$PATH" \
            'traits>=4.6.0' \
            'scipy' \
            'numpy' \
+           'pandas' \
```

Comment:
I am a bit uncertain about this one.  Although indeed pandas is a popular robust way to parse .csv, it is quite a heavy dependency.  Would be useful to know how much (relatively) does it blow up the size of the image?

Reply:
It's 1.599GB without pandas and 1.791GB with pandas. To me that's definitely worth the convenience but totally your call.

Reply:
@daeh could you add it to [`utils/gen-docker-image.sh`](https://github.com/nipy/heudiconv/blob/master/utils/gen-docker-image.sh) as well? We use that script to generate this Dockerfile

Reply:
++ thanks, missed that